src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -111,7 +117,12 @@ PARAM_UROW_IN("labels", "Labels for the training set.", "l");
 
 // Classification options.
 PARAM_MATRIX_IN("test", "Test dataset.", "T");
+/* 
+* The PARAM_UROW_OUT("output") is depracated and
```

Comment:
`deprecated` :)  Also, this block looks like it would all fit on one line.

Reply:
Sorry for the typo